benchmarks/nsinker/README.md

```diff
@@ -0,0 +1,15 @@
+The NSinker benchmark
+=====================
+
+This is the "NSinker" benchmark defined in
+
+Weighted BFBT Preconditioner for Stokes Flow Problems with Highly
+Heterogeneous Viscosity
+Johann Rudi, Georg Stadler, Omar Ghattas
+https://doi.org/10.1137/16M108450X
```

Comment:
Make this a block quote so that the previous sentence doesn't seem to end in [rest of the sentence purposefully omitted]
```suggestion
> Weighted BFBT Preconditioner for Stokes Flow Problems with Highly
> Heterogeneous Viscosity
> Johann Rudi, Georg Stadler, Omar Ghattas
> https://doi.org/10.1137/16M108450X
```